TypeCobol/Compiler/Nodes/Data.cs

```diff
@@ -16,38 +16,79 @@ namespace TypeCobol.Compiler.Nodes {
     public class DataDivision: GenericNode<DataDivisionHeader>, Parent<DataSection> {
 
         public const string NODE_ID = "data-division";
+
+        // The 5 (optional) data sections
+        private FileSection _fileSection;
```

Comment:
This can be simplified using properties with private setter

Reply:
No longer relevant (sections are stored in a array)